modBoot/src/org/aion/Aion.java

```diff
@@ -74,7 +73,9 @@ public static void main(String args[]) {
         CfgAion cfg = CfgAion.inst();
         if (args != null && args.length > 0) {
             int ret = new Cli().call(args, cfg);
-            System.exit(ret);
+            if (!args[0].matches("-n|--network|-d|--datadir")) {
+                exit(ret);
```

Comment:
currently this exit is used for preventing the kernel from executing when there is a cli argument entered; for instance ./aion.sh -h would not run the kernel...maybe i could have it exit when the user inputs an invalid network/datadir instead of letting it run on default parameters

Reply:
thats true, i will change the return integer